.github/workflows/test.yml

```diff
@@ -288,34 +288,8 @@ jobs:
           TURBO_REMOTE_ONLY: true
         run: pnpm -- turbo run run-example -- "${{ matrix.example }}" "${{ matrix.manager }}"
 
-  rust_prepare:
-    name: Check rust crates
-    runs-on: ubuntu-latest-16-core-oss
-    needs: determine_jobs
-    if: needs.determine_jobs.outputs.rust == 'true'
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Setup Rust
-        uses: actions-rs/toolchain@v1
-
-      - name: Setup Go
-        uses: ./.github/actions/setup-go
-        with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-
-      - name: Rust Cache
-        uses: Swatinem/rust-cache@v2
-
-      - name: Run cargo check
-        uses: actions-rs/cargo@v1
-        with:
-          command: check
-          args: --workspace --all-targets
-
```

Comment:
I'm not a fan of removing this job. The idea of this job is to quickly check if the PR is compile-clean, before triggering more expensive jobs. It will save CI cost and report build errors more quickly.

Reply:
But it also costs us CI time

Reply:
reverted this for now